nodes/viz/viewer_texture.py

```diff
@@ -123,11 +123,14 @@ class SvTextureViewerDirSelect(bpy.types.Operator, SvGenericDirectorySelector):
     {
         if (ColorMode) {
            fragColor = texture(image, texCoord_interp);
-        }else{
+        } else {
            fragColor = texture(image, texCoord_interp).rrrr;
         }
     }
 '''
+
+# why (( )) ? see uniform_bool(name, seq) in
+# https://docs.blender.org/api/blender2.8/gpu.types.html
 cMode = ((False,))
```

Comment:
I think that is not necessary anymore

Reply:
i mean cMode variable here

Reply:
yep. i'll move the comment back down,

Reply:
yeah, that global declaration is redundant. but the comment is something i want to keep :)

Reply:
yes, the comment is ok, i was refering only to the global declaration.